projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -171,7 +181,18 @@ export class IgxDatePickerComponent implements IDatePicker, ControlValueAccessor
      * By default the visibility is set to true.
      * @example
      * <igx-date-picker [labelVisibility]="false"></igx-date-picker>
+     * @deprecated Use
+     * ````html
+     * <igx-date-picker>
+     *      <label igxLabel>Custom label</label>
+     * </igx-date-picker>
+     * ````
+     * to set a custom label.
      */
+    @DeprecateProperty(`Deprecated. Use
```

Comment:
I really don't see how that replaces the visibility functionality. Perhaps this property shouldn't be deprecated yet?